src/cli/commands/refs-local.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const { print } = require('../utils')
+
+module.exports = {
+  command: 'refs-local',
+
+  describe: 'List all local references.',
+
+  handler ({ getIpfs, resolve }) {
+    resolve((async () => {
+      const ipfs = await getIpfs()
+      const refs = await ipfs.refs.local()
```

Comment:
Not sure if this got lost - can we use the streaming version for both these commands so we don't forget to update when we fix the streaming in core?

Reply:
Good catch - I had updated the core component but not CLI. Fixed with the latest commit.